wimhofbreather.py

```diff
@@ -0,0 +1,80 @@
+import vlc
+import time
+import datetime
+import os
+import sys
+import random
+
+
+class WimHofBreather:
+    testing_mode = True
+    breathing_times = []
+    random.seed(datetime.datetime.now())
+
+    def __init__(self):
+        print("Alright, guys!")
+        self.breathing_sets = int(input("How many breathing sets would you like to do?\n"))
+        self.script_dir = os.path.dirname(__file__)
+        if len(sys.argv) > 1 and sys.argv[1] == "debug":
+            self.testing_mode = True
+
+    def get_breathing_times(self):
+        return self.breathing_times
+
+    def lets_do_some_breathing(self):
+
+        for index in range(1, self.breathing_sets + 1):
```

Comment:
rename `index` to `iteration` or `round`

Reply:
done